src/ocamlorg_frontend/pages/releases.eml

```diff
@@ -13,69 +13,74 @@ Layout.render
 </div>
 <div class="bg-default dark:bg-dark-default">
     <div class="py-10 lg:py-28">
-        <div class="container-fluid">
+        <div class="container-fluid min-w-full">
             <div class="flex justify-between items-center flex-col md:flex-row">
                 <% let release_num = match List.length releases with 0 -> "No Release" | 1 -> "1 Release" | l -> Printf.sprintf "%i Releases" l in %>
-                <h5 class="font-bold mb-5 md:mb-0"><%s release_num %></h5>
-                <form action="<%s Url.releases %>" method="GET">
-                    <%s! Forms.search_input
+                    <h5 class="font-bold mb-5 md:mb-0">
+                        <%s release_num %>
+                    </h5>
+                    <form action="<%s Url.releases %>" method="GET">
+                        <%s! Forms.search_input
                         ~name:"q"
                         ~label:"Search for a release"
                         ~button_attrs:{|type="submit"|}
                         ""
                         %>
-                </form>
+                    </form>
             </div>
             <div class="overflow-x-scroll lg:overflow-hidden mt-5 lg:mt-10">
                 <% (match List.length releases with | 0 -> %>
-                <div class="flex flex-col items-center justify-center pt-24 space-y-4 text-lighter">
-                    <div>
-                        <%s! Icons.magnifying_glass "h-10 w-10" %>
+                    <div class="flex flex-col items-center justify-center pt-24 space-y-4 text-lighter">
+                        <div>
+                            <%s! Icons.magnifying_glass "h-10 w-10" %>
+                        </div>
+                        <div class="text-xl">No Releases found matching "
+                            <%s search %>"</div>
                     </div>
-                    <div class="text-xl">No Releases found matching "<%s search %>"</div>
-                </div>
```

Comment:
indentation change here makes it hard to see what is changed. Please avoid introducing unnecessary whitespace changes to make reviewing easier.